controllers/apply/form-router-next/lib/form-model.js

```diff
@@ -86,7 +86,7 @@ class FormModel {
 
             function sectionStatus() {
                 const fieldNames = fieldsForSection().map(f => f.name);
-                const fieldData = pick(validation.value, fieldNames);
+                const fieldData = pick(data, fieldNames);
```

Comment:
Using `validation.value` here was preventing some conditional messages from showing, we want to use the original data here.